code/src/index.css

```diff
@@ -11,3 +13,75 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.App{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+form{
+  margin-top: 2rem;
+  background-color: #F2F0F0;
+  padding: 1rem;
+  border: 1px solid black;
+  box-shadow: 8px 8px;
+  margin-bottom: 20px;
+}
+
+#newInput{
+  width: 90%;
+  height: 4rem;
+  border: 1px solid lightgray;
+}
+
+.send-btn{
+  background-color: #FFADAD;
+  border: none;
+  font-weight: bold;
+  border-radius: 2rem;
+  padding:15px;
+}
+
+.warning{
+  margin-left: 1rem;
+}
+
+.all-post{
```

Comment:
i believe you need to add some media queries for styling for smaller screens, the thought boxes are floating outside the screen :)